index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Joseph Kausi'Portfoliot</title>
+</head>
+<body>
+    <header id="main-head" class="family">
+        <h1>WELCOME</h1>
+        <a href=""><img src="image/Icon-Menu.png" alt="Icon-Menu" id="icn-menu"></a>
+    </header>
+    <section class="about-me family">
+        <h2 class="me">Hey there. I’m Mariane
```

Comment:
- [ ] This should be an `h1` element because it is semantically correct and for SEO reasons. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.